geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/tables/AttributeTableV5.scala

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0 which
+ * accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ */
+
+
+package org.locationtech.geomesa.accumulo.data.tables
+
+import com.google.common.collect.ImmutableSortedSet
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.accumulo.core.client.admin.TableOperations
+import org.apache.accumulo.core.conf.Property
+import org.apache.accumulo.core.data.Mutation
+import org.apache.hadoop.io.Text
+import org.locationtech.geomesa.accumulo.data.AccumuloFeatureWriter.{FeatureToMutations, FeatureToWrite}
+import org.locationtech.geomesa.accumulo.data._
+import org.locationtech.geomesa.utils.geotools.RichAttributeDescriptors.RichAttributeDescriptor
+import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+import org.locationtech.geomesa.utils.stats.IndexCoverage
+import org.opengis.feature.`type`.AttributeDescriptor
+import org.opengis.feature.simple.SimpleFeatureType
+
+import scala.util.Try
+
+/**
+ * Contains logic for converting between accumulo and geotools for the attribute index
+ */
+@deprecated
+object AttributeTableV5 extends GeoMesaTable with Logging {
```

Comment:
this is the old attribute table code - i haven't changed anything here, just renamed the file.